src/openforms/registrations/contrib/objects_api/tests/test_backend.py

```diff
@@ -564,19 +600,261 @@ def test_submission_with_objects_api_backend_override_defaults_do_not_upload_csv
+                        "attribute": RegistrationAttribute.initiator_geslachtsnaam,
+                    },
+                },
+                {
+                    "key": "tussenvoegsel",
+                    "type": "textfield",
+                    "registration": {
+                        "attribute": RegistrationAttribute.initiator_tussenvoegsel,
+                    },
+                },
+                {
+                    "key": "geboortedatum",
+                    "type": "date",
+                    "registration": {
+                        "attribute": RegistrationAttribute.initiator_geboortedatum,
+                    },
+                },
+                {
+                    "key": "coordinaat",
+                    "type": "map",
+                    "registration": {
+                        "attribute": RegistrationAttribute.locatie_coordinaat,
+                    },
+                },
+            ],
+            submitted_data={
+                "voornaam": "Foo",
+                "achternaam": "Bar",
+                "tussenvoegsel": "de",
+                "geboortedatum": "2000-12-31",
+                "coordinaat": [52.36673378967122, 4.893164274470299],
+            },
+            language_code="en",
+        )
+        submission_step = submission.steps[0]
+        step_slug = submission_step.form_step.form_definition.slug
+
+        mock_service_oas_get(m, "https://objecten.nl/api/v1/", "objecten")
+        mock_service_oas_get(m, "https://documenten.nl/api/v1/", "documenten")
+
+        objects_form_options = dict(
+            objecttype="https://objecttypen.nl/api/v1/objecttypes/2",
+            objecttype_version=2,
+            productaanvraag_type="testproduct",
+            informatieobjecttype_submission_report="https://catalogi.nl/api/v1/informatieobjecttypen/2",
+            upload_submission_csv=True,
+            informatieobjecttype_submission_csv="https://catalogi.nl/api/v1/informatieobjecttypen/5",
+            organisatie_rsin="123456782",
+            content_json="""
+            {\n
+                "bron": {\n
+                    "naam": "Open Formulieren",\n
+                    "kenmerk": "{{ submission.kenmerk }}"\n
+                },\n
+                "type": "{{ productaanvraag_type }}",\n
+                "aanvraaggegevens": {% json_summary %},\n
+                "taal": "{{ submission.language_code  }}"\n
+            }""",
+            zaak_vertrouwelijkheidaanduiding="geheim",
+            doc_vertrouwelijkheidaanduiding="geheim",
+        )
+
+        expected_result = {
+            "url": "https://objecten.nl/api/v1/objects/1",
+            "uuid": "095be615-a8ad-4c33-8e9c-c7612fbf6c9f",
+            "type": objects_form_options["objecttype"],
+            "record": {
+                "index": 0,
+                "typeVersion": objects_form_options["objecttype_version"],
                 "data": {
                     "data": {
                         f"{step_slug}": {
```

Comment:
`f"{variable}"` is the same as `variable`

see also the earlier remark about setting up your test data with a known slug so you can avoid using variables in your assertions